maven/lib/dependabot/maven/file_fetcher.rb

```diff
@@ -25,13 +25,21 @@ def fetch_files
         fetched_files << pom
         fetched_files += child_poms
         fetched_files += relative_path_parents(fetched_files)
+        fetched_files << extensions unless extensions.nil?
```

Comment:
Might be clearer as an `if`
```suggestion
        fetched_files << extensions if extensions
```